src/tir/transforms/lower_tvm_builtin.cc

```diff
@@ -247,7 +247,7 @@ class BuiltinLower : public StmtExprMutator {
     Array<PrimExpr> packed_args = {op->args[0], stack_value_, stack_tcode_,
                                    ConstInt32(arg_stack_begin),
                                    ConstInt32(arg_stack_begin + op->args.size() - 1)};
-    return Call(DataType::Int(32), builtin::tvm_call_packed_lowered(), packed_args);
+    return Call(op->dtype, builtin::tvm_call_packed_lowered(), packed_args);
```

Comment:
the return value of tvm_call_packed_lowered is int32, and the return value is passed through the return value argument instead of the return type, so it is supposed to be i32